user.html

```diff
@@ -0,0 +1,242 @@
+    <script src="https://kit.fontawesome.com/9ce739b2b2.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="./css/base.css" />
+    <link rel="stylesheet" type="text/css" href="./css/user.css"/>
+    <title>Freedom</title>
+    <link rel="shortcut icon" type="image/x-icon" href="./photos/favicon.ico" />
+    <script type="text/javascript" src="admin.js"></script>
+  </head>
+  <body>
+    <div id="wrapper">
+      <div id="header">
+        <nav class="navbar navbar-expand-md mb-4 bg-light">
+         <img src="./photos/favicon.ico" alt="Peace Icon" id="peace-icon">
+         <div class="toogle-button">
+           <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+             <i class="fa fa-bars"></i>
+           </button>
+         </div>
+         <div class="collapse navbar-collapse" id="collapsibleNavbar">
+           <ul class="navbar-nav mr-auto">
+             <li class="nav-item active">
+               <a class="nav-link" href="#">Home</a>
+             </li>
+             <li class="nav-item">
+               <a class="nav-link" href="#">Contact</a>
+             </li>
+           </ul>
+           <div class="nav-link">
+             <a class="nav-avatar" href="#">
+               <img src="./photos/avatar.jpg" alt="Avatar" class="avatar">
+             </a>
+             <a class="nav-username" href="#"> Username </a>
+           </div>
+         </div>
+      </nav>
+    </div>
+
+    <div id="content">
+          <div class="container tittle-line">
+            <div class="row">
+              <div class="col-sm-2">
+                <img  id="UN-logo" src="./photos/un.png">
+              </div>
+              <div class="col-sm-2">
+                <div id="page-header-separator">
+                <a href="https://www.un.org/en/" id="UN-tittle"> United<br>Nations </a>
+              </div>
+            </div>
+            <div class="col-8">
+              <p id="header-description">  Promote peaceful and inclusive societies for sustainable development,
+                 provide access to justice for all and build effective, accountable and inclusive institutions at all levels</p>
+            </div>
+          </div>
+        </div>
+
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-3 column-1">
+            <div class="card card-id">
+              <div class="gradient">
+                <img alt="user-icon" src="./photos/avatar.jpg" class="card-img-top">
```

Comment:
Some ids/classes are `like-this` and some are `likeThis`. Decide one naming style and go with it.